Update docs to show spin in system path (#301)
fermyon/developer: content/cloud/quickstart.md

curl -fsSL https://developer.fermyon.com/downloads/install.sh | bash
```

This command downloads and unpacks the latest Spin binary in the current directory.

> **It's highly recommended to add Spin to a folder, which is on your path**, e.g.:

Reviewer: I don't know about _highly_; it works fine if it isn't in your PATH (as long as you remember where it _is_). Also suggest a little rewording:

```suggestion
**It's recommended to add Spin to a directory in your `$PATH`**, e.g.:
```

Author: Thanks @lann
